__test__/wasm.spec.ts

```diff
@@ -28,7 +28,7 @@ test('em box', (t) => {
 
 test('glyph path to_string()', (t) => {
   const fontkit = new FontKit();
-  const key = fontkit.add_font_from_buffer(fontData!);
+  const [key] = fontkit.add_font_from_buffer(fontData!);
```

Comment:
https://github.com/alibaba/font-toolkit#fontkitadd_font_from_bufferbuffer-uint8array---fontkey
looks like array (`Vec[FontKey]` in rust) is the type actually returned, which part should be updated to be consistent?